packages/common/test/deserialization.test.ts

```diff
@@ -122,3 +123,17 @@ test('Expired transactions can be deserialized', () => {
         new TransactionExpiry(new Date(2000, 1), true)
     );
 });
+
+test('Receive return value can be deserialized', () => {
```

Comment:
I made another test (to test this :upside_down_face:), but I won't bother trying to push it to your fork...
```
import * as fs from 'fs';
...
test('Return value can be deserialized - auction', () => {
    const returnValue = deserializeReturnValue(
        Buffer.from(
            '00000b0000004120676f6f64206974656d00a4fbca84010000',
            'hex'
        ),
        Buffer.from(
            fs.readFileSync('./test/resources/auction-with-errors-schema.bin')
        ),
        'auction',
        'view'
    );

    expect(returnValue.item).toEqual('A good item');
    expect(returnValue.end).toEqual('2022-12-01T00:00:00+00:00');
    expect(returnValue.auction_state).toHaveProperty('NotSoldYet');
    expect(returnValue.highest_bidder).toHaveProperty('None');
});

```